com.github.nrittsti.NTag.yml

```diff
@@ -0,0 +1,32 @@
+app-id: com.github.nrittsti.NTag
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+command: ntag
+finish-args:
+  - --socket=x11
+  - --socket=wayland
+  - --filesystem=host
```

Comment:
It's something easy to change for users, either via Flatseal or `flatpak override`. I'd change it to `xdg-music`, `xdg-download`, `/media` and `/run/media`.

Reply:
okay is done so